editor/components/EditableField.tsx

```diff
@@ -1,25 +1,19 @@
-import { useState } from "react";
+import { useForm, RegisterOptions } from "react-hook-form";
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
 import { InputType } from "@/types/types";
 import { LookupTableOption } from "@/types/relationships";
 
-interface DataCardInputProps {
-  /**
-   * The initial value to populate the input
-   */
+
+
+interface EditableFieldProps {
+  /** The initial value to populate the input */
```

Comment:
yay i think making the comments one line is great to save on space

Reply:
i like it!